### PR TITLE
scenarios selected from list

### PR DESCRIPTION
**Pull request by Shikib**
Scenarios for main task (not onboarding or "pre-ready" description) are selected from a list.

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @dakshvar22 will take a look at it as soon as possible ✨

File: parlai/mturk/tasks/woz/scenarios/book_ride_v0.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "User": {
+      "completion_requirements": ["You have booked a ride."],
+      "completion_questions": [
+        "...find you a suitable ride?",
+        "...stay calm and helpful throughout the dialogue?"
+      ],
+      "task_description": "You want a ride from Forbes and Murray to Broadway and Main. Your name is Mark."
+    },
+    "Wizard": {
+      "task_description": "AA ...",
```
Comment (JEM-Mosig) [inline parlai/mturk/tasks/woz/scenarios/book_ride_v0.json]: It seems like this has to be changed, too?

Comment (Shikib) [reply parlai/mturk/tasks/woz/scenarios/book_ride_v0.json]: This is automatically generated from the template file. What's wrong with it?

Comment (JEM-Mosig) [reply parlai/mturk/tasks/woz/scenarios/book_ride_v0.json]: The task description should not be "AA ...". But if it is generated, I guess we can adjust the template in a separate PR.

**Review by JEM-Mosig — APPROVED** [review]

**Pull request merged**